contracts/exchange/ExampleERC20.sol

```diff
@@ -0,0 +1,19 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.24;
+pragma abicoder v2;
+
+import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+
+contract ExampleERC20 is ERC20 {
+    uint8 _decimals = 18;
```

Comment:
You shouldn't create such storage variable since original ERC20 doesn't have it. You can just override the function like this:
`function decimals() public view override returns (uint8) {
        return 18;
    }`